packages/config/config/devices/0x010f/templates/fibaro_template.json

```
@@ -0,0 +1,224 @@
+{
+	"default_metadata_for_fibaro": {
+		"inclusion": "Turn the primary controller of Z-Wave network into inclusion mode, quickly press key S1 or B-button on product's housing three times",
+		"exclusion": "Turn the primary controller of Z-Wave network into exclusion mode, quickly press key S1 or B-button on product's housing three times",
+		"reset": "1. Power-cycle the device\n2. Press and hold the B-button to enter the menu\n3. Wait for the LED to turn yellow\n4. Quickly release and click B-button again\n5. Device will restart and LED will turn red"
+	},
+	"send_secure_commands_group2": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Send Secure Z-wave Commands to 2nd Association Group",
```

```suggestion
		"label": "Send Secure Commands to 2nd Association Group",
```
Seems unnecessary. Please do the same change to the other variants of this param.

---

Done